src/App.tsx

```diff
@@ -110,7 +117,12 @@ function App({ classes, history }: AppProps) {
       </Drawer>
       <main className={classes.content}>
         <div className={classes.toolbar} />
-        <Route exact={true} path="/" component={ListSwaps} />
+        <Route
+          exact={true}
+          path="/"
+          render={() => <Redirect to="/show_resource/swaps" />}
+        />
+        <Route path="/show_resource" component={ShowResource} />
```

Comment:
Not sure if this helps with your reload-problem but:

You can specify path-variables in these routes. https://reacttraining.com/react-router/web/example/url-params

Reply:
I'm not sure how to use that to express that a re-render should happen when the url goes from `localhost:8000/show_resource/swaps` to `localhost:8000/show_resource/swaps/rfc003/:id`.

Reply:
I was more thinking of using a path variable for the resource link:

`<Route path="/show_resource/:resourceLink">`